src/hermes/commands/harvest/git.py

```diff
@@ -312,15 +312,10 @@ def harvest_git(click_ctx: click.Context, ctx: HermesHarvestContext):
 
     _audit_contributors(git_contributors, logging.getLogger('audit.git'))
 
-    ctx.update_from({
-        '@context': [
-            "https://doi.org/10.5063/schema/codemeta-2.0",
-            {'hermes': 'https://software-metadata.pub/ns/hermes/'}
-        ],
-
-        '@type': "SoftwareSourceCode",
-        'contributor': [contributor.to_codemeta() for contributor in git_contributors._all],
-    }, git_branch=git_branch)
+    ctx.update('contributor',
+               [contributor.to_codemeta() for contributor in git_contributors._all],
+               git_branch=git_branch)
+    ctx.add_context(('hermes', 'https://software-metadata.pub/ns/hermes/'))
```

Comment:
Adding the hermes context isn't really git harvester-specific. I think this should happen elsewhere. Maybe the hermes context should exist by default in all HermesContext objects.

Reply:
In this specific case, yes. However, I think all harvesters should be able to add arbitrary contexts.

Reply:
Absolutely 👍🏻

Reply:
Will move adding the hermes context to the (tada) `HermesContext` and chuck the URL into a constant.

Reply:
Cool 👍🏻

I don't think removing the context is necessary, either. (If it was, every context that was added would need to be checked, not only the hermes context.) Maybe at some point in the future we could have a custom write function that does this before writing the data to disk. But before doing this we should look into some json-ld libraries...